### PR TITLE
protosync v0.5.4

### PR DESCRIPTION
**Pull request by orf**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by chenrui333** [comment]
```
  protosync
    * Formula license ["MIT"] does not match GitHub license ["Apache-2.0"].
    * GitHub repository not notable enough (<30 forks, <30 watchers and <75 stars)
```

File: Formula/p/protosync.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class Protosync < Formula
+  desc "Synchronise remote .proto files to a local directory"
+  homepage "https://github.com/cashapp/protosync"
+  url "https://github.com/cashapp/protosync/archive/refs/tags/v0.5.4.tar.gz"
+  sha256 "2ef679764b687bb79fd897aebaf70796cb6df8560fbb122d780da628b6abd5c9"
+  license "MIT"
```
Comment (chenrui333) [inline Formula/p/protosync.rb:6]:
```suggestion
  license "Apache-2.0"
```

File: Formula/p/protosync.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(output: bin/"protosync"), "./cmd/protosync"
```
Comment (chenrui333) [inline Formula/p/protosync.rb:11]:
```suggestion
    system "go", "build", *std_go_args(ldflags: "-s -w"), "./cmd/protosync"
```

File: Formula/p/protosync.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  end
+
+  test do
+    assert_match(/^Usage: protosync/, shell_output("#{bin}/protosync --help"))
```
Comment (chenrui333) [inline Formula/p/protosync.rb:15]: need a more substantial test

**Comment by SMillerDev** [comment]
Hi, this seems like a nice software, but not notable enough for us to package in Homebrew for now. This is not a decision on the merits of the software, but we cannot distribute every single piece of software and have to draw a line on “notability” somewhere. Of course, we would be happy to reconsider in the future.

Please consider hosting it in a personal tap, which is very easy to do: https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap

**Comment by orf** [comment]
No worries! Makes sense, thank you for the review.

**Pull request closed without merging**